meeting-notes/2018/2018-07-02--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,204 @@
+ - Next:
+   - GraphSync
+   - More IPLD Format implementation changes
+
+@hugomrdias
+ - Done:
+    - Q2 retro and Q3 planning
+    - Learn about the modules i'm lead maintainer
+    - Make proper CI tests for large files
+    - js.ipfs.io review
+ - Blocked:
+ - Next:
+   - Finish CI tests, publish and deploy
+   - Work on PRs/Issues from multihashing and multipart
+   - Q3 planning
+
+@vasco-santos
+ - Done:
+   - Continue IPNS PR
+     - https://github.com/ipfs/js-ipfs/pull/1400
+     - Added nanoseconds precision on IPNS record validity
+     - Added key parameter for publish
+     - Added recursive parameter for resolve
+   - Released js-ipns initial version
+     - https://github.com/ipfs/js-ipns/pull/1
+   - Created PR with an initial version of the IPNS specs
+     - https://github.com/ipfs/specs/pull/184
+     - Some points still need to be written and some diagrams need to be created
+   - PRs on libp2p modules for changing lodash imports
+     - https://github.com/libp2p/js-libp2p-connection-manager/pull/14
+     - https://github.com/libp2p/js-libp2p-floodsub/pull/56
+     - https://github.com/libp2p/js-libp2p-keychain/pull/24
+     - https://github.com/libp2p/js-libp2p-bootstrap/pull/78
+   - Renamed js-libp2p-railing to js-libp2p-bootstrap
+   - libp2p / ipfs Q3 OKRs
+ - Blocked:
+   - N/A
+ - Next:
+   - Get IPNS working local PR ready for the final reviews
+     - Finish remaining optional params
+     - Logging / errors refactor
+     - Finish tests
+     - Add lru cache 
+   - Continue writting ipIPNS spec
+   - Help freezing libp2p OKRs
+   
+@fsdiogo
+ - Done:
+   - Getting up to speed with the WebUI and Companion codebase
+   - Did a bunch of PR reviews to check the current work
+   - Did some small PRs with some fixes to both WebUI and Companion
+ - Blocked:
+   - N/A
+ - Next:
+   - Implement the Peers page on WebUI revamp
+
+@gar
+ - Done:
+ - Blocked:
+   - toPrint change https://github.com/libp2p/js-peer-id/pull/81
+ - Next:
+   - cli tests sharing ipfs instances
+   - ipfs bitswap ledger (cli, core, and API)
+     - blocked by libp2p PR above
+   - ipfs bitswap reprovide implementation
+
+@achingbrain
+ - Done:
+   - adding logging to ipfsd-ctll 
+   - adding logging to js-api
+   - resurrected npm-on-ipfs project
+ - Blocked:
+   - https://github.com/ipfs/go-ipfs/pull/5139
+   - https://github.com/ipfs/js-ipfs/pull/1360
+   - https://github.com/ipfs/aegir/pull/245
+   - https://github.com/ipfs/npm-go-ipfs-dep/pull/28
+   - https://github.com/ipfs/interop/pull/22
+ - Next:
+   - Fix up tests after Alan's interface test refactor
+   - Merge MFS
+   - Flesh out interop tests
+   - Continue with npm-on-ipfs
+
+@travisperson
+  - Next:
+    - Working on standardizing test formats https://github.com/ipfs/interface-ipfs-core/issues/313
+
+@mkg20001
+ - Done
+    - aegir check if deps defined in source code are in package.json or if there are unused deps in package.json: https://github.com/ipfs/aegir/pull/241
+
+@name
```

Comment:
@mikeal wanna add your update here as well?

Reply:
definitely, but i don't have write access to this repo :)

i can wait till it's merged and send a PR or you can give me write access :)

Reply:
added you :)